src/functionalTest/resources/application.conf

```diff
@@ -14,8 +14,9 @@ idam-users-cmc-username=${IDAM_USERS_CMC_USERNAME}
 idam-users-cmc-password=${IDAM_USERS_CMC_PASSWORD}
 idam-users-div-username=${IDAM_USERS_DIV_USERNAME}
 idam-users-div-password=${IDAM_USERS_DIV_PASSWORD}
-idam-users-finrem-username=${IDAM_USERS_FINREM_USERNAME}
-idam-users-finrem-password=${IDAM_USERS_FINREM_PASSWORD}
+#use divorce for finrem
+idam-users-finrem-username=${IDAM_USERS_DIV_USERNAME}
+idam-users-finrem-password=${IDAM_USERS_DIV_PASSWORD}
```

Comment:
Instead we can remove finrem config as users are at jurisdiction level

Reply:
true. values are still sticked in enum individually